### PR TITLE
Update release pipeline to get storage account details

### PR DESCRIPTION
**Pull request by namadabu**
## Description
Update release pipeline to get storage account details. We deploy storage accounts as part of our PR environment. We will be using these storage accounts to export data to. Since we also need to validate the exported data, we need access to the storage account from within the test code. We will use the existing keyvault to store the required storage account details.

## Related issues
Addresses [issue [AB#73559](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/73559)].

## Testing
Describe how this change was tested.

### CODE REVIEW

File: release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1 [hunk 1]
```diff
@@ -109,6 +109,15 @@ function Add-AadTestAuthEnvironment {
         $application = Get-AzureAdApplicationByIdentifierUri $fhirServiceAudience
     }
 
+    Write-Host "Getting storage accounts in resource group"
+    $storageAccounts = Get-AzStorageAccount -ResourceGroupName $ResourceGroupName
+    foreacH ($storageAccount in $storageAccounts) {
```
Comment (brendankowitz) [inline release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1:114]: `foreacH` casing

File: release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1 [hunk 2]
```diff
@@ -109,6 +109,15 @@ function Add-AadTestAuthEnvironment {
         $application = Get-AzureAdApplicationByIdentifierUri $fhirServiceAudience
     }
 
+    Write-Host "Getting storage accounts in resource group"
```
Comment (brendankowitz) [inline release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1:112]: Is this just used for testing?

Comment (namadabu) [reply release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1:112]: Yes. These storage accounts are deployed with the PR resource group and then torn down with that environment. We need access to the storage accounts from the test code in order to read the exported data.

File: release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1 (hunk 1 above)
Comment (namadabu) [reply release/scripts/PowerShell/FhirServerRelease/Public/Add-AadTestAuthEnvironment.ps1]: Fixed

**Pull request merged**